features/deleting.feature

```diff
@@ -0,0 +1,22 @@
+Feature: As a visitor,
+  I would like to delete emails,
+  that are receiven
+
+  Background:
+    Given there is users named:
```

Comment:
Isn't there already a step like `Given the following users exist`? Reuse that step instead of writing new ones.